openedx/core/lib/xblock_utils/__init__.py

```diff
@@ -215,8 +214,7 @@ def wrap_xblock_aside(
     }
 
     if hasattr(frag, 'json_init_args') and frag.json_init_args is not None:
-        # Replace / with \/ so that "</script>" in the data won't break things.
-        template_context['js_init_parameters'] = json.dumps(frag.json_init_args).replace("/", r"\/")
```

Comment:
@noraiz-anwar Why is this not needed anymore?

Reply:
We are now using a standard way of dumping string to json. 
See https://github.com/edx/edx-platform/pull/19309/files#diff-280961e2fb974649372c302d00a71decR8